gamemodes/horde/entities/effects/medic_flame.lua

```diff
@@ -32,18 +32,18 @@ function EFFECT:Init(data)
 			end)
 		end
 
-		for i = 1, 2 do
-			local p = self.Emitter:Add("particles/smokey", Startpos)
-			p:SetDieTime(1.5)
-			p:SetStartAlpha(50)
-			p:SetEndAlpha(0)
-			p:SetStartSize(math.Rand(2, 4))
-			p:SetEndSize(math.random(70, 90))
-			p:SetRoll(math.random(-10, 10))
-			p:SetRollDelta(math.random(-10, 10))
-			p:SetVelocity(((Hitpos - Startpos):GetNormal() * math.random(500, 800)) + VectorRand() * math.random(1, 60) + Vector(0,0,20))
-			p:SetCollide(true)
-			p:SetColor(40, 40, 40)
+		for e = 1, 2 do
+			local g = self.Emitter:Add("particles/smokey", Startpos)
```

Comment:
instead of g this can be something like particle,  no need for 1 char vars. it just makes the code harder to read.